bundle/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml

```diff
@@ -337,7 +423,7 @@ spec:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.namespace
-                image: mongodb/mongodb-atlas-kubernetes-operator:0.5.0
+                image: quay.io/jianrzha/mongodb-atlas-operator:0.5.37
```

Comment:
can we change this to using org instead of using a personal account?

Reply:
+1, if we need to host the image and commit a link to it, let's move to our team org (you should already have access): 
https://quay.io/organization/ecosystem-appeng

Reply:
Agreed. Will do.